$NetBSD: patch-src_corelib_configure.json,v 1.1 2019/03/26 11:14:16 adam Exp $

Revert change from 5.12.1 to correctly detect 64-bit atomics and dlopen().

--- src/corelib/configure.json.orig	2019-03-06 07:59:50.000000000 +0000
+++ src/corelib/configure.json
@@ -133,6 +133,7 @@
         "libatomic": {
             "label": "64 bit atomics",
             "test": {
+                "include": [ "atomic", "cstdint" ],
                 "tail": [
                     "void test(volatile std::atomic<std::int64_t> &a)",
                     "{",
@@ -151,7 +152,6 @@
                 ],
                 "qmake": "CONFIG += c++11"
             },
-            "headers": [ "atomic", "cstdint" ],
             "sources": [
                 "",
                 "-latomic"
@@ -160,13 +160,13 @@
         "libdl": {
             "label": "dlopen()",
             "test": {
+                "include": "dlfcn.h",
                 "main": [
                     "dlclose(dlopen(0, 0));",
                     "dlsym(RTLD_DEFAULT, 0);",
                     "dlerror();"
                 ]
             },
-            "headers": "dlfcn.h",
             "sources": [
                 "",
                 "-ldl"
